### ISSUE TITLE
Add filter to add custom developers to verified list

### ISSUE THREAD

**Issue by smileBeda** [TukuToi/tukutoi-cp-directory-integration#12]
It would be nice to have a filter so a plugin developer can add itself to the verified list of developer... _organizations_.
Not single users

**Comment by smileBeda** [comment]
This is a security issue.

If the plugin offers such filter, then another plugin can hook into that filter, and no matter how we would implement that filter, the third party plugin could pass _any_ repository to the array of repos we pull, with or without the user knowing.

For example:
- we could make it so that if a third party registers via filter, we save this new repo in the options. The user would then know it, _if_ they visit the settings page. Yet, they would still not be able to remove it, nor would they actually be sure that this is a vetted repo, and they did _not_ decide themselves to add it.
- if we just use a filter (without saving it in the database) the user will not even know that the third party plugin added itself (or ANY other repo really) to the array, and our directory plugin would happily pull whatever is passed.

_This is exactly what the below feature is intended for, and thus, this needs no filter, but actual action of the third party developer to get listed:_
>By default, there is a vetted list of Organizations added to the plugin. If a Developer wants to appear on said list, they can submit a PR to the `github-orgs.txt` File of **this Plugin**, by **adding their Guthub Organization data to the JSON array**. >**The Organization AND the PR initiator will be reviewed both by the author of this plugin as well the ClassicPress Plugin Review Team.** Only after careful assessment the Developer will be added to the Verified List of Organizations, and thus appear pre-selected in the Repositories queried by this plugin.

Closing as won't do.
